src/parse.coffee

```diff
@@ -33,8 +33,7 @@ parse = (source, callback) ->
       err = new Error(err.message)
 
     if apiElements
-      apiElements = apiElements.toRefract()
-      apiElements.content = apiElements.content.concat(annotations)
+      apiElements.content.unshift(warning) if warning
```

Comment:
I discovered that the problem isn't necessarily with uptowns class implementation, it is because of the way it is used in minim. Since a `BaseElement` knows about a namespace/registry it is wrapped in a function which passes that namespace/registry:

```js
function (registry) {
  return createClass({}); // BaseElement
}
```

There is a circular reference between a meta BaseElement and the namespace. Creating a namespace, creates a new meta class (BaseElement) in the context of the namespace/registry. The element knows about the namespace because the element model itself handles deserialisation.

[refract.py](https://github.com/kylef/refract.py) goes for a completely different architecture so that the `Element` has no idea about "serialisation" or "namespace" concepts and it's just a "dumb model". There is a separation of concerns between the serialisations and the element itself.